docs/docs/dev_docs/tutorials/token_portal/typescript_glue_code.md

```diff
@@ -47,8 +40,8 @@ Open `cross_chain_messaging.test.ts` and paste the initial description of the te
 
 ```typescript
 import { expect, jest} from '@jest/globals'
-import { AccountWallet, AztecAddress, DebugLogger, EthAddress, Fr, computeAuthWitMessageHash, createDebugLogger, createPXEClient, waitForPXE } from '@aztec/aztec.js';
-import { getInitialTestAccountsWallets } from '@aztec/accounts/testing';
```

Comment:
Just a heads up, I the name of this function will change back after the next release.

Reply:
oh. that's tomorrow then - maybe we should just keep it